File: replay-fx/src/Models/Ball.js
const Ball = props => ({
		timeOnGround: props.positionalTendencies.timeOnGround,
		timeLowInAir: props.positionalTendencies.timeLowInAir,
		timeHighInAir: props.positionalTendencies.timeHighInAir,
		timeInDefendingHalf: props.positionalTendencies.timeInDefendingHalf,
		timeInAttackingHalf: props.positionalTendencies.timeInAttackingHalf,
		timeInDefendingThird: props.positionalTendencies.timeInDefendingThird,
		timeInNeutralThird: props.positionalTendencies.timeInNeutralThird,
		timeInAttackingThird: props.positionalTendencies.timeInAttackingThird,
		timeBehindBall: props.positionalTendencies.timeBehindBall,
		timeInFrontBall: props.positionalTendencies.timeInFrontBall,
		timeNearWall: props.positionalTendencies.timeNearWall,
		timeInCorner: props.positionalTendencies.timeInCorner,
		timeOnWall: props.positionalTendencies.timeOnWall,
	  averageSpeed: props.averages.averageSpeed,
})

export default Ball;

File: ReplayFx/Models/toBeConverted/Ball.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Transactions;

namespace ReplayFx.Models.Data
{
    class Ball
    {
        public double timeOnGround { get; set; }
        public double timeLowInAir { get; set; }
        public double timeHighInAir { get; set; }
        public double timeInDefendingHalf { get; set; }
        public double timeInAttackingHalf { get; set; }
        public double timeInNeutralThird { get; set; }
        public double timeInAttackingThird { get; set; }
        public double timeInDefendingThird { get; set; }
        public double timeNearWall { get; set; }
        public double timeInCorner { get; set; }
        public double timeOnWall { get; set; }
        public double averageSpeed { get; set; }
        public double neutralPosessionTime { get; set; }
    }
}
